dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/dsprops/Properties_postgresql.java

```diff
@@ -0,0 +1,254 @@
+/*******************************************************************************
```

Comment:
How did you manufacture this class?  Did you copy another one and change the names to Postgres as needed?

Reply:
Added in the fields manually.  Then, used eclipse tools to generate getters, setters, and toString method.  Then added in the annotations and (unfortunately) had to edit the toString method to look like the other properties classes.

Reply:
Thank you!